controller/proxy-injector/webhook.go

```diff
@@ -108,6 +109,9 @@ func Inject(api *k8s.API,
 
 	if parent != nil {
 		recorder.Event(*parent, v1.EventTypeNormal, eventTypeInjected, "Linkerd sidecar proxy injected")
+		if report.TracingEnabled {
```

Comment:
Should we have a disabled event as well? I'm not 100% sure it is needed.

Reply:
Just like inject/skip, I think it will be easier for users to determine if tracing is enabled by looking at events than the proxy injector logs. Happy to remove if you don't think it's needed.

Reply:
Sorry, completely misread your comment. Yeah, I agree we don't need to add `disable` events, as I like to reserve events for unusual stuff.